packs/data/spell-effects.db/spell-effect-blessing-of-defiance-2-action.json

```diff
@@ -1,92 +0,0 @@
-{
-    "_id": "w9HpSwBLByNyRXvi",
-    "img": "systems/pf2e/icons/spells/blessing-of-defiance.webp",
-    "name": "Spell Effect: Blessing of Defiance (2 Action)",
```

Comment:
If we delete this we will need to write a migration for it. It's fine to restore this whole thing and just leave it unused, adding it to the list of condensed effects to migrate away later #4144